applications/nrf_desktop/doc/swift_pair_app.rst

```diff
@@ -0,0 +1,51 @@
+.. _nrf_desktop_swift_pair_app:
+
+Swift Pair module
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Swift Pair module is used to enable or disable the Swift Pair Bluetooth advertising payload depending on the selected Bluetooth peer (used local identity).
+The module distinguishes between the dongle peer and the general Bluetooth peers.
+
+Module events
+*************
+
+.. include:: event_propagation.rst
+    :start-after: table_swift_pair_app_start
+    :end-before: table_swift_pair_app_end
+
+.. note::
+    |nrf_desktop_module_event_note|
+
+Configuration
+*************
+
+To use the Swift Pair module, you must enable the following Kconfig options:
+
+* :kconfig:option:`CONFIG_BT_ADV_PROV_SWIFT_PAIR` - The nRF Desktop's :ref:`nrf_desktop_ble_adv` uses the :ref:`bt_le_adv_prov_readme` to generate advertising and scan response data.
+  The Swift Pair advertising data provider (:kconfig:option:`CONFIG_BT_ADV_PROV_SWIFT_PAIR`) is used to add the Swift Pair payload to the advertising data.
```

Comment:
```suggestion
  To add the Swift Pair payload to the advertising data, use the Swift Pair advertising data provider (:kconfig:option:`CONFIG_BT_ADV_PROV_SWIFT_PAIR`).
```

Reply:
The new wording may be a bit counterintuitive I think. The module automatically uses the Swift Pair advertising data provider. User does not use the API of the advertising data provider.

Reply:
Agree. Please ignore this comment.